### DISCUSSION TITLE
[MAYA-113125] export constrained objects

### DISCUSSION THREAD

**Discussion by sanders3d** [Autodesk/maya-usd#1631] [General]
I'm not being able to export even the simplest animation. Isn't constrained animation supported?
In the example I've selected the "cube_grp" and exported it to usd with animation enabled.
![image](https://user-images.githubusercontent.com/44543707/129696486-f8d5f683-1b97-40a0-9632-d8ce94da4533.png)

I'm quite surprised how limited the USD functionality is at this point. Or.. I might be doing everything wrong at this point

**Comment by sanders3d** [comment]
actually, this worked. But not through file - export selected

```
import maya.cmds as mc

mc.mayaUSDExport(
    file="C:/temp/test_script.usd",
    selection=True,
    exportDisplayColor=True,
    shadingMode=None,
    frameRange=( 1.0, 120.0),
    mergeTransformAndShape=True,
    verbose=True,
)
```

**Comment by wtelford** [comment]
Hey Stefan,
Nice catch. Looks like you found an inconsistency between our UI and command-based export behavior. Please log this as an issue.
